articles/azure-monitor/app/opentelemetry-overview.md

```diff
@@ -21,6 +21,9 @@ There are two methods to instrument your application:
 
 **Autoinstrumentation** enables telemetry collection through configuration without touching the application's code. Although it's more convenient, it tends to be less configurable. It's also not available in all languages. See [Autoinstrumentation supported environments and languages](codeless-overview.md). When autoinstrumentation is available, it's the easiest way to enable Azure Monitor Application Insights.
 
+> [!TIP]
+> Currently, [AAD Auth](azure-ad-authentication.md) is not available with autoinstrumentation. If you require AAD Auth, you'll need to use manual instrumentation.
```

Comment:
i didn't realize that we introduce this new term `autoinstrumentation`, but i can easily find `auto-instrumentation` elsewhere. e.g. https://learn.microsoft.com/en-us/troubleshoot/azure/azure-monitor/app-insights/auto-instrumentation-troubleshoot. 

should we be consistent everywhere? one way or the other not mixed.

Reply:
previous comment has nothing to do with this change though. but i think we should have a follow-up PR to fix the inconsistencies. auto-instrumentation is what we used in the java space. would like to keep it forward. new term might seem fancy at first, but it will cause confusion.

Reply:
FYI, @AaronMaxwell see comment from helen...